godot_communicator.py

```diff
@@ -50,15 +54,17 @@ def godot_sender(udp_ip, udp_port, track):
             continue
         except socket.timeout:
             # drive car. sleep amount is inversely dependant of velocity
-            velocity = calculate.velocity(velocity, 0)
+            velocity = calculate.velocity(velocity, time_stamp, throttle_force=0)
+            time_stamp = time.time()
             radius = calculate.radius(x[i], x[i - 1], x[i - 2], y[i], y[i - 1], y[i - 2])
             centripetal_force = calculate.centripetal_force(velocity, radius)
 
             print("F: %s" % centripetal_force)
 
             if calculate.is_derailed(centripetal_force):
-                text = "derailed"
-            elif velocity == 0:
+                #text = "derailed"
+                sock.sendto("derailed".encode('utf-8'), addr)
+            elif velocity <= 3:
```

Comment:
This code looks very strange to me but I assume we will work to improve this in a different task, right?